Source/UIResources_Localization/en.lproj/LimeAuth.strings

```diff
@@ -92,6 +92,13 @@
 "limeauth.auth.retypeNewPassword"           = "Retype new password";
 "limeauth.auth.passwordsNotMatch"           = "Passwords did not match. Try again.";
 
+// Passphrase strength
+"limeauth.auth.passStrengthWarning"         = "Warning";
+"limeauth.auth.passStrengthDifferentPin"    = "Use different";
+"limeauth.auth.passStrengthDifferentPassword" = "Use different";
+"limeauth.auth.passStrengthIgnorePin"       = "Continue";
+"limeauth.auth.passStrengthIgnorePassword"  = "Continue";
```

Comment:
I suggest following strings:
```
 "limeauth.auth.passStrengthDifferentPin"    = "Change passcode";
 "limeauth.auth.passStrengthDifferentPassword" = "Change password";
 "limeauth.auth.passStrengthIgnorePin"       = "Use anyway";
 "limeauth.auth.passStrengthIgnorePassword"  = "Use anyway";
```

Reply:
Updated

Reply:
The rest of string table is using term "passcode" for PIN.